src/key_value_store.js

```diff
@@ -408,13 +409,26 @@ export class KeyValueStoreLocal {
         checkParamOrThrow(index, 'index', 'Number');
 
         const files = await readdirPromised(this.localStoragePath);
-        let keys = files.map(file => path.parse(file).name).sort(); // Array is sorted to emulate API.
+        let keys = [];
+        for (const file of files) {
```

Comment:
Here is problem. If some of the files gets deleted between `readdirPromised` and `statPromised` then the latter one throws a file not found error. I am not sure how to handle this. Maybe we could just catch that type of error and remove that file from array.